governance/RELEASE-MANAGEMENT.md

```diff
@@ -0,0 +1,78 @@
+# Release Management
+
+This document describes ORAS project release management, which includes release versioning, supported releases, and supported upgrades. This document is applicable to all ORAS sub-projects.
+
+## Glossary of Terms
+
+- **X.Y.Z** refers to the version (based on git tag) of ORAS sub-project that is released. 
+- **Breaking changes** refer to schema changes, flag changes, and behavior changes of ORAS sub-projects that may require existing content to be upgraded and may also introduce changes that could break backward compatibility.
```

Comment:
I want to call out that breaking changes to experimental features are not considered as breaking changes.